core/consensus/babe/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "substrate-consensus-babe"
+version = "1.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+description = "BABE consensus algorithm for substrate"
+edition = "2018"
+
+[dependencies]
+parity-codec = "3.4"
+parity-codec-derive = "3.3"
+babe_primitives = { package = "substrate-consensus-babe-primitives", path = "primitives" }
+primitives = { package = "substrate-primitives", path = "../../primitives" }
+runtime_support = { package = "srml-support", path = "../../../srml/support" }
+runtime_version = { package = "sr-version", path = "../../sr-version" }
+runtime_io = { package = "sr-io", path = "../../sr-io" }
+inherents = { package = "substrate-inherents", path = "../../inherents" }
+srml-consensus = { path = "../../../srml/consensus" }
+substrate-telemetry = { path = "../../telemetry" }
+client = { package = "substrate-client", path = "../../client" }
+consensus_common = { package = "substrate-consensus-common", path = "../common" }
+authorities = { package = "substrate-consensus-authorities", path = "../authorities" }
+slots = { package = "substrate-consensus-slots", path = "../slots"  }
+runtime_primitives = { package = "sr-primitives", path = "../../sr-primitives" }
+futures = "^0.1.26"
```

Comment:
isn't `^` implied in just using the version number? Or is there any other particular reason these dependencies have them (but the ones on the top don't)? Just asking to learn, if they are the same, I don't think it needs to be change.

Reply:
@gnunicorn It is, actually.  I didn’t know that until I looked it up.